backend/src/main/java/com/github/nickklock/backend/models/Spot.java

```diff
@@ -0,0 +1,23 @@
+package com.github.nickklock.backend.models;
+
+import org.springframework.data.annotation.Id;
+
+import java.util.List;
+
```

Comment:
Forgot to add the document here, as you can see i tested the this only with my written tests 😄